src/tactic/lint/frontend.lean

```diff
@@ -70,6 +70,20 @@ meta def get_checks (slow : bool) (extra : list name) (use_only : bool) :
   let default := if slow then default else default.filter (λ l, l.2.is_fast),
   list.append default <$> get_linters extra
 
+/--
+`xs.to_chunks n` splits the list into sublists of size at most `n`,
+such that `(xs.to_chunks n).join = xs`.
+-/
+meta def list.to_chunks {α} (n : ℕ) : list α → list (list α)
```

Comment:
Worth moving these to `data.list.defs`?

Reply:
Possibly worth merging first, so that all future CI runs, including the one that moves these defs, are shorter

Reply:
I'm not sure what you mean by "including the one that moves these defs", since each commit to this branch [can already take advantage of the shorter CI times](https://github.com/leanprover-community/mathlib/runs/1927494934).

Reply:
Whoops, that was a dumb claim